x-pack/plugins/fleet/common/types/models/agent_policy.ts

```diff
@@ -33,7 +33,6 @@ export interface NewAgentPolicy {
   fleet_server_host_id?: string | null;
   schema_version?: string;
   agent_features?: Array<{ name: string; enabled: boolean }>;
-  is_protected?: boolean;
```

Comment:
This was optional, why are we removing this?

Reply:
same as above